spanner/jdbc/src/test/java/com/example/spanner/JdbcSampleIT.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.spanner.DatabaseAdminClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.Collections;
+import java.util.UUID;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Unit tests for {@code JdbcSample} */
+@RunWith(JUnit4.class)
+public class JdbcSampleIT {
+  // The instance needs to exist for tests to pass.
+  private final String instanceId = System.getProperty("spanner.test.instance");
+  private final String databaseId = formatForTest(System.getProperty("spanner.sample.database"));
```

Comment:
Do we need to add something in Java-Docs-Samples-Testing for this?

Reply:
No, AFAICT this should work out of the box. This is also modeled to work exactly as the existing `SpannerSampleIT`: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/c3c4600b0848a48dece316cf809f95721c427f44/spanner/cloud-client/src/test/java/com/example/spanner/SpannerSampleIT.java#L39-L40